test/runtests.jl

```diff
@@ -5,7 +5,8 @@ using Test
 @testset "IndexedDims.jl" begin
     doctest(IndexedDims)
 
-    a = IndexedDimsArray([1 2 3; 4 5 6; 7 8 9], 2:4, 2:4)
+    data = [1 2 3; 4 5 6; 7 8 9]
+    a = IndexedDimsArray(data, 2:4, 2:4)
 
     # constructor
```

Comment:
Why not using nested testsets?

Reply:
no good reason. would be a nice follow-up PR https://github.com/invenia/IndexedDims.jl/issues/10